web_src/js/components/ViewFileTreeItem.vue

```diff
@@ -68,6 +72,7 @@ const doGotoSubModule = () => {
     :class="{'selected': selectedItem === item.fullPath}"
     :title="item.entryName"
     @click.stop="doLoadFileContent"
+    @click.middle.stop="doLoadFileContent"
```

Comment:
Everything is handled in `@click.stop`, so the `@click.middle.stop` should be removed?